packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -557,6 +542,55 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     }
   }
 
+  /// Returns the index of the first character after extent.
+  ///
+  /// Setting includeWhitespace to false will only return the index of non-space
+  /// characters.
+  @visibleForTesting
+  static int nextCharacter(int extent, String string, [bool includeWhitespace = true]) {
+    if (extent >= string.length) {
+      return string.length;
+    }
+
+    int count = 0;
+    final Characters remaining = string.characters.skipWhile((String currentString) {
+      if (count <= extent) {
+        count += currentString.length;
+        return true;
+      }
+      if (includeWhitespace) {
+        return false;
+      }
+      return _isWhitespace(currentString.characters.first.toString().codeUnitAt(0));
```

Comment:
That line is equivalent to:
```dart 
   return _isWhitespace(currentString.codeUnitAt(0));
```

Reply:
Ah thanks, good call!

Reply:
This will be fixed when I reland this PR in https://github.com/flutter/flutter/pull/59778.